aws/resource_aws_launch_template_test.go

```diff
@@ -525,6 +527,8 @@ resource "aws_launch_template" "test" {
 
   network_interfaces {
     network_interface_id = "${aws_network_interface.test.id}"
+    ipv4_address_count = 2
```

Comment:
We should probably split these out into their own acceptance tests and test configurations to make them more maintainable in the future. 👍